src/useCases/drive/deleteShareLink.ts

```diff
@@ -8,8 +8,7 @@ import { sharedLinksUpdated } from './getShareLink';
 
 export const deleteShareLink = async ({ shareId, type }: { shareId: string; type: 'file' | 'folder' }) => {
   try {
-    const result = await drive.share.deleteShareLink({ shareId });
-
+    await drive.share.stopSharingItem({ itemUUID: shareId, itemType: type });
```

Comment:
so shareId is actually the item uuid? I'm having a hard time understanding variable names here

Reply:
No, I just didn't finish checking the changes I made and it remained unchanged.
I change it.